steamos-waydroid-installer.sh

```diff
@@ -311,6 +329,7 @@ else
 	fi
 
 	# casualsnek script
+	echo -e "$current_password\n" | sudo -S pacman -Sy --noconfirm lzip
```

Comment:
```suggestion
```

afaict, `lzip` is [only needed when installing `gapps`](https://github.com/casualsnek/waydroid_script/blob/1a2d3ad643206ad5f040e0155bb7ab86c0430365/stuff/gapps.py#L126) with the `casualsnek` script